Sources/NIOCore/ChannelHandlers.swift

```diff
@@ -68,8 +68,9 @@ public final class AcceptBackoffHandler: ChannelDuplexHandler, RemovableChannelH
                     scheduleRead(at: self.nextReadDeadlineNS!, context: context)
                 }
             }
+        } else {
```

Comment:
I'm inclined to make this behaviour configurable. Users may be relying on the existing reporting, which makes sense as a default behaviour, but for the use-case we have it may well be sensible to suppress the error when it is "handled".